pkg/operator/csi/credentialsrequestcontroller/credentials_request_controller.go

```diff
@@ -69,9 +78,19 @@ func (c CredentialsRequestController) sync(ctx context.Context, syncContext fact
 	if apierrors.IsNotFound(err) {
 		return nil
 	}
+
 	if err != nil {
 		return err
 	}
+	clusterCloudCredential, err := c.typedVersionedOperatorClient.OperatorV1().CloudCredentials().Get(ctx, clusterCloudCredentialName, metav1.GetOptions{})
```

Comment:
this needs to be from a lister so that on a standard sync loop you do you not hit the kube-apiserver.

Once this is wired from a lister, that looks fine.

Reply:
okay... though we do not have a lister for `CredentialRequest` type - so it was hitting kube-apiserver anyways on every standard sync.I think I could fix this in a follow up or this PR.